tests/functional/utils/S3UtilsMongoClient.js

```diff
@@ -451,7 +464,26 @@ describe('S3UtilsMongoClient::getObjectMDStats', () => {
                             logger,
                             (err, data) => {
                                 assert.deepStrictEqual(err, null);
-                                assert.deepStrictEqual(data, expected);
+                                // assert.deepStrictEqual(data, expected);\
```

Comment:
commented-out code

Reply:
It was meant to be removed in favor of the below lines, so I removed it.